coffer.cabal

```diff
@@ -22,6 +22,8 @@ library
       Backend
```

Comment:
I've found 2 bugs so far:

-----

Creating an entry with a multiline field content succeeds (as in, it creates a file `/tmp/pass-store/dir/entry4` on disk), but then `coffer view` says it doesn't exist.

```
$ coffer create /dir/entry4 --field "user=$(echo -e "first\nsecond")"
[SUCCESS] Entry created at '/dir/entry4'.

$ coffer view /dir                                                   
[ERROR] Entry or directory not found at '/dir'.
```

Haven't looked too much into it, but I suspect the parsing is failing. And the parser failure is being masked by the other bug I mentioned in another comment; we're returning `Nothing` in `pbReadSecret` when decoding fails.



-----

This may or may not be a bug, maybe I'm doing something wrong? I don't know. But setting up a new `pass` instance and then running `coffer /` fails for me. Running `coffer view` with any path other than `/` works though:

```
$ export PASSWORD_STORE_DIR='/tmp/pass-store'

$ pass init dc@diogocastro.com
mkdir: created directory '/tmp/pass-store/'
Password store initialized for dc@diogocastro.com

$ coffer /
ListSecrets: Path {unPath = []}
out: Just [".gp"]
OtherError "Internal error:\nBackend returned a secret that is not a valid entry or directory name.\nGot: '.gp'.\n"
```
-----